docs_src/microservices/device/supported/device-onvif-camera/supplementary-info/ONVIF-protocol.md

```diff
@@ -4,19 +4,52 @@
 This Onvif Camera Device Service is developed to control/communicate ONVIF-compliant cameras accessible via http in an EdgeX deployment
 
 ## Table of Contents
-
-- [Onvif Features](#onvif-features)  
+- [API Specifications](#device-onvif-api-specifications)  
+- [ONVIF Protocol Properties](#onvif-protocol-properties)  
+- [ONVIF Features](#onvif-features)  
 - [Custom Features](#custom-features)  
 - [How does the service work?](#how-does-the-device-service-work)  
 - [Tested Onvif Cameras](#tested-onvif-cameras)
 
-## OpenAPI Spec
+## Device ONVIF API Specifications
 The latest version {{version}} of the device service API specifications can be found
 [here](../swagger.md).
 
 
+## ONVIF Device Service Protocol Properties
+### ONVIF Protocol
+
+All properties in the `ONVIF` protocol field are defined by internal device information and some user defined information.
```

Comment:
The protocol name is defined as `Onvif` currently, and I believe it is case-sensitive.